lib/rack/lint.rb

```diff
@@ -39,6 +35,12 @@ class LintError < RuntimeError; end
     ##
     ## A Rack application is a Ruby object (not a class) that
     ## responds to +call+.
+    def initialize(app)
+      raise LintError, "app must respond to call" unless app.respond_to?(:call)
```

Comment:
You don't need to move the definition for things to work, though it's not a problem to move it, just more code churn.

Reply:
I understand your concern. The reason for the move is so that the specification documentation and the actual check are proximal.